lib/ocamlorg_web/graphql.mli

```diff
@@ -25,10 +25,21 @@ val get_packages_result
   -> string option
   -> Package.t list
   -> packages_result
+(** [get_packages_result total_packages offset limit startswith packages]
```

Comment:
One last comment and this looks good to go -- to me it seems a little odd that we're passing in `total_packages` into this function. I don't think it is used in any way right, just passed directly to the record? 

Unless I'm mistaken (please tell me if I am) everywhere that passes in `total_packages` is just computing `List.length packages` e.g. https://github.com/ocaml/v3.ocaml.org-server/blob/b23e68833617ec80df6b4fe5911f7863af1217d9/lib/ocamlorg_web/graphql.ml#L276

Could this not be done inside the function so the user of the API doesn't have to do that work ? Maybe I'm missing something?

Reply:
It was used to match the limit 
```
let limit =
  match limit with None -> total_packages | Some limit -> limit
in

Reply:
Ah okay makes sense 👍  The only comment I'd have about it is that we have some untested logic then. Your test that mimics what goes on here https://github.com/ocaml/v3.ocaml.org-server/pull/73/files#diff-c55503669a514a15d5842bfbe254058410c19d81d5fca26dc13dbd0545cb1030R103-R114 only captures some of the logic that is used in the resolver for example the following aren't tested:
 - What if the `limit` is `None`?
 - What if the offset is not `0`?
 - Are the correct `3` packages returned here ?
 
 I'm happy to merge this and have this be a follow-up PR/issue, let me know your thoughts :)) ?